lib/iris/cube.py

```diff
@@ -1742,7 +1742,7 @@ def data(self, value):
             self._numpy_array = value
 
     def has_lazy_data(self):
-        return True if self._numpy_array is None else False
```

Comment:
What? `self._numpy_array is None` is already a logical test, so return the result of the logical test.